examples/sample-app/version-1/kustomization.yaml

```diff
@@ -1,10 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
 resources:
   - app-pre-deploy.yaml
   - app-pre-deploy-eval.yaml
   - ../base
 
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
 patches:
   - path: appcontext.yaml
-  - path: manifest.yaml
+  - patch: |-
```

Comment:
why there needs to be an appcontext.yaml file and additionally we patch the file?

Reply:
because we download the original podtato head manifest in the base so kustomize needs to override it with our stuff, in example one we put the predep tasks and the v 0.1.1 in example 2 we do the same but also bump the version of frontend to v0.1.2, I let the context in the two version for future flexibility if we want to introduce better showcasing of our features there... I think it is out of scope of this fix but in my mind we should at least have an example with different contexts 🤷‍♀️